src/Symfony/Component/Form/Tests/Extension/DataCollector/FormDataExtractorTest.php

```diff
@@ -394,13 +397,35 @@ public function testExtractViewVariables()
             'id' => 'foo_bar',
             'name' => 'bar',
             'view_vars' => array(
-                'a' => "'bar'",
-                'b' => "'foo'",
-                'c' => "'baz'",
-                'id' => "'foo_bar'",
-                'name' => "'bar'",
+                'a' => '"bar"',
+                'b' => '"foo"',
+                'c' => '"baz"',
+                'id' => '"foo_bar"',
+                'name' => '"bar"',
             ),
-        ), $this->dataExtractor->extractViewVariables($view));
+        ), $this->inlineData($this->dataExtractor->extractViewVariables($view)));
+    }
+
+    private function inlineData(array $extraction)
+    {
+        $h = fopen('php://memory', 'r+b');
+        $dumper = new CliDumper($h);
+        $inlined = array();
+
+        foreach ($extraction as $k => $v) {
+            if (is_array($v)) {
+                $inlined[$k] = $this->inlineData($v);
+            } elseif ($v instanceof Data) {
+                $dumper->dump($v->withRefHandles(false));
```

Comment:
We can't use `true` as second arg without bumping the minimum required version. Not worth it.